README.md

```diff
@@ -83,6 +83,24 @@ This package uses benchee to benchmark code. To run the benchmarker:
 
     $ MIX_ENV=bench mix run bench/<file>.exs
 
+### AddressSanitizer
+
+A memory testing setup is included to detect memory errors in the NIF.
+It's set up in a Docker container to ensure reproducability.
+
+To run the tests, build the container, which will build a version of the NIF with AddressSanitizer enabled.
+Then, run it with an =APPSIGNAL_PUSH_API_KEY= and =APPSIGNAL_APP_NAME= set to ensure AppSignal is enabled, and to be able to verify that data appears in AppSignal after running the test:
+
+    docker build -t appsignal-elixir-asan .
```

Comment:
```suggestion
    docker build --platform linux/amd64 -t appsignal-elixir-asan .
```
Based on the release architecture of the agent.tar.gz file you want to make sure this is an x86_64 image.